### PR TITLE
Translate "OOP" to Ukrainian

### PR DESCRIPTION
**Pull request by Herobread**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Oleksii Nazarenko

## Language: 

- Ukrainian

## Terms defined:

- OOP

### CODE REVIEW

**Review by Dod900ls1 — APPROVED** [review]
LGTM

File: glossary.yml [hunk 1]
```diff
@@ -7675,6 +7675,13 @@
       Um paradigma de programação no qual dados (atributos) e funções (métodos)
       são encapsulados em objetos que interagem entre si por meio de
       interfaces bem definidas.
+  uk:
+    term: "об'єктно-орієнтоване програмування"
+    acronym: "ООП"
+    def: >
+      Стиль програмування, в якому функції та дані об'єднуются разом в
```
Comment (olexandr-konovalov) [inline glossary.yml:7682]: об'єднуються  - з мʼяким знаком

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]

File: glossary.yml (hunk 1 above)
Comment (Herobread) [reply glossary.yml:7682]: @olexandr-konovalov updated

**Comment by olexandr-konovalov** [comment]
@Herobread looks like you forgot `git add ...` before amending your commit though - can't see any changes

**Comment by Herobread** [comment]
@olexandr-konovalov , yes, I think now it works

File: glossary.yml [hunk 2]
```diff
@@ -7675,6 +7675,13 @@
       Um paradigma de programação no qual dados (atributos) e funções (métodos)
       são encapsulados em objetos que interagem entre si por meio de
       interfaces bem definidas.
+  uk:
+    term: "об'єктно-орієнтоване програмування"
+    acronym: "ООП"
+    def: >
+      Стиль програмування, в якому функції та дані об'єднуються разом в 
+      [об'єкти](#object), які взяємодіють між собою тільки через
```
Comment (olexandr-konovalov) [inline glossary.yml:7683]: взяємодіють -> взAємодіють

Oops, missed this when read it before, sorry !

Comment (Herobread) [reply glossary.yml]: updated

Comment (olexandr-konovalov) [reply glossary.yml]: Nice - at least that's some additional practice in editing a PR :)

**Review by olexandr-konovalov — APPROVED** [review]
@elletjies please merge

**Pull request merged**